src/modules/search/components/Track.jsx

```diff
@@ -5,12 +5,22 @@ import classname from 'classnames';
 export default function Track({onClick, playing, search, thumbnail, title}) {
 	return (
 		<StyledResult onClick={onClick} className={classname({playing})}>
-			<ImageHolder search={search}>
+			<ImageHolder className="image-holder" search={search}>
 				<img src={thumbnail} />
-				<PlayBackground />
-				{search ? <AddPlus /> : <PlayButton />}
+				<PlayBackground className="play-background" />
+				{search ? <AddPlus /> : <PlayButton className="play-button" />}
 			</ImageHolder>
 			<h4>{title}</h4>
+			<MenuTrigger className="menu-trigger">
```

Comment:
@jamesmcnamara can you help me turn this into a Click event as opposed to a hover, see line 165